scripts/booksHandler.js

```diff
@@ -1,36 +0,0 @@
-import { GOOGLE_URL } from "./variables.js";
-import { createCard } from "./card.js";
-
-//function to get a book object with some information to display
-export const getBookData = (book) => {
-  let checkAuthors = book.volumeInfo.authors;
-  if (checkAuthors.length > 2) {
-    checkAuthors = `${checkAuthors[0]} et al.`;
-  }
-
-  // https://loremflickr.com/g/300/200/dog - image placeholder
-
-  return {
-    image: (book.volumeInfo.imageLinks.thumbnail ?? "https://loremflickr.com/g/300/200/dog" ), // test this line
```

Comment:
This is a little fragile and is breaking on some searches. [Optional Chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) Would be helpful here. Something like 
```js
book?.volumeInfo?.imageLinks?.thumbnail ?? "https://loremflickr.com/g/300/200/dog"
```